src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.CodeAnalysis
     internal partial class ErrorLogger : IDisposable
     {
         // Internal for testing purposes.
-        internal const string OutputFormatVersion = "0.1";
+        internal const string OutputFormatVersion = "0.4";
```

Comment:
Are we emitting the Roslyn diagnostic related locations information? If not, see the way binskim.exe handles this.
Also note that the IL checker persists the assembly as the analysis target, rather than the source file. We might prefer to have the roslyn compilation-driven analysis use resultFile for the literal result and put something else in the analysisTarget (such as a project file? kind getting out there now). this would add more repeated data to the log file, of course.